crates/speedy-transform/src/wasm_types.rs

```diff
@@ -0,0 +1,41 @@
+use serde::{Deserialize, Serialize};
+
+pub struct TransformOutput {
+  pub code: String,
+  pub map: Option<String>,
+}
+
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct TransformConfig {
+  pub remove_use_effect: Option<bool>,
+  pub react_runtime: Option<bool>,
+  pub babel_import: Option<Vec<BabelImportConfig>>,
+}
+
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct BabelImportConfig {
+  pub from_source: String,
+  pub replace_css: Option<RepalceCssConfig>,
+  pub replace_js: Option<RepalceSpecConfig>,
```

Comment:
Any reason for naming `RepalceSpecConfig` instead of `ReplaceJsConfig`? It would be better for keeping consistency.

Reply:
all use `ReplaceJsConfig`